### DISCUSSION TITLE
Alternative way for latest version daisyui in node js version ≤ 16

### DISCUSSION THREAD

**Discussion by JunedSetiawan** [saadeghi/daisyui#2015] [General]
When installing the latest version of DaisyUI on node js version 16 and below you will get an error                    
      
[plugin:vite:css] [postcss] Object.hasOwn is not a function

If you don't want or can't update the node version for some reason.

An alternative way to get rid of the error is to replace the Object.hasOwn() method with the Object.hasOwnProperty() method.

And it works on node js version 16 and below .
There may be other alternatives..

**Comment by calebbehayilu** [comment]
where should i replace that?

**Comment by saadeghi** [comment]
Why don't you upgrade the node?

**Reply by JunedSetiawan** [reply]
I'm still using Windows 7 and Windows 7 doesn't support the latest version of Node JS, actually I want to switch to Linux but I'm still unsure

**Reply by saadeghi** [reply]
Windows 7 is from 14 years ago.  
Maybe this trick or or other tricks (search them) work for you:  
https://stackoverflow.com/questions/70756105/how-to-latest-version-of-node-js-on-windows-7
